.travis.yml

```diff
@@ -1,5 +1,8 @@
 sudo: false
 language: node_js
+addons:
+    code_climate:
+        repo_token: $CODECLIMATE_REPO_TOKEN
```

Comment:
where is this variable set?

codeclimate checks the code quality (with linters), but the code is instrumented and subject to code coverage in every repo, as we are calculating it in travis. then codeclimate will show the last one published.

what i suggest is then also to instrument the code and publish it only on the master branch; i will modify travis.sh accordingly if you are ok.